plugins/pulp_rpm/plugins/distributors/export_distributor/export_utils.py

```diff
@@ -118,23 +119,20 @@ def validate_export_config(config):
 
 def create_date_range_filter(config):
     """
-    Create a date filter based on start and end issue dates specified in the repo config. The
-    returned
-    filter is a dictionary which can be used directly in a mongo query.
+    Create a date filter based on start and end issue dates specified in the repo config.
 
     :param config: plugin configuration instance; the proposed repo configuration is found within
     :type  config: pulp.plugins.config.PluginCallConfiguration
 
-    :return: date filter dict with issued date ranges in mongo query format
-    :rtype:  {}
+    :return: Q object with start and/or end dates
+    :rtype:  mongoengine.Q
     """
     start_date = config.get(constants.START_DATE_KEYWORD)
     end_date = config.get(constants.END_DATE_KEYWORD)
-    date_filter = None
+
     if start_date and end_date:
-        date_filter = {ASSOCIATED_UNIT_DATE_KEYWORD: {"$gte": start_date, "$lte": end_date}}
```

Comment:
if start_date and end_date both evaluate to False this function used to return None, but now it won't return None in any case. I think I'm reading this right... If I am that should be fixed and the docstring should indicate it can return 'mongoengine.Q or None'